scripts/utils.sh

```diff
@@ -141,7 +139,7 @@ function close_external_ports() {
 
 function add_firewalld_port() {
     port=$1
-    if [ "${EXTERNAL_PORT}" = "y" ]; then
+    if [ "${EXTERNAL_PORT}" ]; then
```

Comment:
```
$ EXTERNAL_PORT=false
$ if [ "$EXTERNAL_PORT" ]; then
∙ echo "foo"
∙ fi
foo
```
this condition evaluates to true even though EXTERNAL_PORT is set to false. If it intended?

Reply:
yes, you are correct it because of the brackets.
fixed